frontend/portals/MaintenanceMode/index.jsx

```diff
@@ -79,30 +113,40 @@ class MaintenanceMode extends Component {
               onTouchStart={this.handleTouchStart}
               onTouchEnd={this.handleTouchEnd}
             />
-            <button onClick={this.handleClick}>
+            <button type="button" onClick={this.handleClick}>
               <img className={styles.image} src={imageSource} alt={appConfig.shopName} />
             </button>
           </div>
         </div>
       ) : (
-        <div className={styles.background} >
+        <div className={styles.background}>
           <div className={styles.container}>
             <img className={styles.image} src={appConfig.logo} alt={appConfig.shopName} />
             <h3 onTouchStart={this.handleTouchStart} onTouchEnd={this.handleTouchEnd}>
               {customHeadline || <I18n.Text string="maintenanceMode.headline.text" />}
             </h3>
             {customMessage || <I18n.Text string="maintenanceMode.message.text" />}
+            {(!isIosDevice && androidLink) && (
+              <Link className={styles.linkButton} href={androidLink} state={{ target: '_blank' }}>
+                <Button>{androidButtonText}</Button>
+              </Link>
+            )}
+            {(isIosDevice && iosLink) && (
+              <Link className={styles.linkButton} href={iosLink} state={{ target: '_blank' }}>
+                <Button>{iosButtonText}</Button>
+              </Link>
+            )}
           </div>
         </div>
       );
     if (
       enableMaintenanceMode &&
       testUser.indexOf(userEmail) === -1 &&
```

Comment:
I know that this isn't yours but I think it would be better to replace 
`testUser.indexOf(userEmail) === -1` with `!testUser.includes(userEmail)`